private/plop/plopfile.js

```diff
@@ -0,0 +1,9 @@
+import { addSkuCommandGenerator } from './generators/add-sku-command/index.js';
+import { addSkuCodemodGenerator } from './generators/add-sku-codemod/index.js';
+
+/** @param {import('plop').NodePlopAPI} plop*/
```

Comment:
The `plop` CLI doesn't support `.ts` files directly, so we'd have to compile it from TypeScript, which I don't think is worth it, so I added JSDoc type annotations instead.